modules/doc/content/sqa/index.md

```diff
@@ -30,3 +30,5 @@ The documentation for the SQA releases can be found using the following links:
 - [v1.0.0](https://mooseframework.org/releases/moose/v1.0.0)
 
 !sqa report
+
```

Comment:
Let's move this up above the Table and make a heading for it. I had to look at the source before I finally found the location of this link. It's pretty buried sitting underneath the table. If you haven't looked at the rendered version of your site. Go back to the PR "Conversation" page an look for the MooseBuild comment. It'll post the link to the rendered version of your page so you can see what it looks like.